R/susie_wrapper.R

```diff
@@ -342,7 +342,7 @@ susie_rss_pipeline = function(sumstat, R, ref_panel, n, L, var_y, QC = TRUE, imp
 #'
 #' @param z Numeric vector of z-scores corresponding to the effect size estimates, with names matching the reference panel's variant IDs.
 #' @param R Numeric matrix representing the LD (linkage disequilibrium) matrix.
-#' @param ref_panel Data frame with 'chrom', 'pos', 'variant_id', 'A0', 'A1' column that matches the names of z.
+#' @param ref_panel Data frame with 'chrom', 'pos', 'variant_id', 'A1', 'A2' column that matches the names of z.
```

Comment:
A1, A2 or A2, A1?

Reply:
THe order in this table (reference panel) will not change the result, so A1 A2 / A2 A1 both works. About the order in the output I prefer A1 A2, since a lot sumstat file put the order of effect allele then other allele